README.md

```diff
@@ -24,9 +24,7 @@ $ pip install yamale
 ```
 
 ### Manual
-1. Download Yamale from: https://github.com/23andMe/Yamale/archive/master.zip
-2. Unzip somewhere temporary
-3. Run `python setup.py install` (may have to prepend `sudo`)
+Run `pip install https://github.com/23andMe/Yamale/archive/master.zip` (may have to prepend `sudo`)
```

Comment:
Thanks for your interest in Yamale and your PR. In this case, manual installation refers to installing without pip. 

If you'd like to add this instruction under the `pip` section as a way to install the latest master branch, it would probably fit better there. Thanks.